builtin/providers/aws/resource_aws_vpc.go

```diff
@@ -55,6 +55,13 @@ func resourceAwsVpc() *schema.Resource {
 				Computed: true,
 			},
 
+			"assign_amazon_ipv6_cidr_block": {
```

Comment:
This name feels a little strange... having read the docs I now understand what you meant by "amazon" here but it reads as if "Amazon IPv6" is some sort of product name.

I must admit I'm not sure what to suggest as an alternative. `assign_generated_ipv6_cidr_block` is one idea that came to mind, though it's long. Perhaps `assign_generated_ipv6_range` is a shorter way of saying it. I also wonder if the word "assign" already has the "chosen by someone else" meaning embedded in it enough, so that `assign_ipv6_range` would be sufficient to get the point across.

Feel free to ignore me, since I know this is not the most constructive comment... :confounded: